its/src/test/java/com/sonar/it/scanner/msbuild/EnvironmentVariable.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SonarScanner for .NET
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonar.it.scanner.msbuild;
+
+public class EnvironmentVariable {
+  private final String name;
+  private final String value;
+
+  public EnvironmentVariable(String name, String value) {
+    this.name = name;
+    this.value = value;
+  }
+
+  public String getName() {
```

Comment:
[education] For sonar-dotnet and the C# part of the code we have a coding style that says properties should go before constructors. Now this is definitely a property or at least it behaves like one but on the other side it is just a method in java. Do we have somewhere documented what order of members do we expect for java code?

Reply:
No, I don't think so. We only have https://github.com/SonarSource/sonar-developer-toolset#code-style

Reply:
Long shot, maybe there should be an LT level document? There is a big chunk of the LT codebase in java and it would be nice to have a consistent coding style in between.

Reply:
_(I asked on slack, however it's a bit out of scope)_